retention/r30.py

```diff
@@ -257,9 +254,22 @@ def mysql_reporter(**kwargs):
     cursor = db.cursor()
 
     def report(date, r30, **client_r30):
```

Comment:
Could you add a docstring to this function describing what each argument is? (It took me a while to figure out what `client_r30` actually was - it appears to be a dictionary of `client_name: count`).

Also it's odd that we deconstruct `client_r30` when passing it to `report`... which just constructs it back into a dict. But outside of the scope of this PR I guess :slightly_smiling_face:

Reply:
Ah, yeah, that's a remnant from when I was only using the CSV DictWriter... the same signature just ended up getting carried over. Doesn't really make sense anymore. Fixing (+ docstring)